api.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"github.com/hashicorp/vault/api"
+)
+
+// Store arbitrary string-data in Vault and return single-use wrapping token
+// Use the Vault response-wrapping features
```

Comment:
godoc style is for these comments to be in the form sentences describing what the thing is. The first sentence should start with the identifier and describe it. So in this case I would write this as:

```
// Store stores arbitrary string-data in Vault and returns a single-use wrapping token
// using the Vault response-wrapping features.
```

It's a slight difference, but that matches the tone I would expect to see in godoc. If you want some examples, you can take a look at the stdlib documentation, which is all generated from godoc comments.